packages/datepickers/src/styled/styled-datepicker.tsx

```diff
@@ -5,44 +5,46 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { css } from 'styled-components';
-import {
-  DEFAULT_THEME,
-  PALETTE,
-  isRtl,
-  retrieveComponentStyles
-} from '@zendeskgarden/react-theming';
-import rgba from 'polished/lib/color/rgba';
+import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 import math from 'polished/lib/math/math';
 
-const retrieveSpacing = ({ isSmall }: { isSmall?: boolean }) => {
+const retrieveSpacing = ({ isSmall, theme }: { isSmall?: boolean } & ThemeProps<DefaultTheme>) => {
```

Comment:
This aligns with the public prop `small` as defined by design when we created the initial API. I don't think making this API differ from the public prop or have a breaking change for this since there isn't a noticeable gain.